gst-libs/mfx/gstmfxencoder.c

```diff
@@ -514,12 +510,12 @@ gst_mfx_encoder_set_frame_info (GstMfxEncoder * encoder)
     priv->params.mfx.FrameInfo.AspectRatioH = priv->info.par_d;
     priv->params.mfx.FrameInfo.Width = GST_ROUND_UP_32 (priv->info.width);
     priv->params.mfx.FrameInfo.Height = GST_ROUND_UP_32 (priv->info.height);
-	
-	gst_mfx_extVideoSignal_from_colorimetry(&GST_VIDEO_INFO_COLORIMETRY
-	(&priv->info), &priv->extVideoSignalInfo);
-
-	priv->extparam_internal[priv->params.NumExtParam++] = (mfxExtBuffer*)&(priv->extVideoSignalInfo);
-
+    
+    gst_mfx_extsig_from_colorimetry(&GST_VIDEO_INFO_COLORIMETRY
```

Comment:
80 character limit!

Reply:
Sorry, not used to formatting my code like this

Reply:
https://gstreamer.freedesktop.org/documentation/frequently-asked-questions/developing.html#what-is-the-coding-style-for-gstreamer-code

Np. It's really only to enforce consistency and readability. But code in itself looks good to me.